test/test_view.c

```diff
@@ -29,6 +29,7 @@ DEALINGS IN THE SOFTWARE.  */
 #include <unistd.h>
 #include <stdlib.h>
 #include <string.h>
+#include <getopt.h>
```

Comment:
I'm working on a simple Windows (MSVC, VS2015) build, so not a regular posix system. I like the VS IDE and I'm curious if I can get it to work without any big changes to the code.

Including getopt.h is a bit of a kludge, but as I saw it done in another c file, I figured it would be an acceptable thing to do. It's not strictly needed though. I can find other ways to work around it.